From 922ff0da833e726054df2fc87abdc201fbc5e21b Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Mon, 9 May 2011 14:11:22 +0200
Subject: [PATCH] Specficy subdevice 0 for front/surround* of HDA-Intel.conf

The subdevice 0 is used for multi-channel outputs.
This fixes the detection of multi-channel devices of VIA codecs
by PulseAudio.

Reported-by: Raymond Yau <superquad.vortex2@gmail.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 src/conf/cards/HDA-Intel.conf |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/conf/cards/HDA-Intel.conf b/src/conf/cards/HDA-Intel.conf
index bc82dbf..d4f2667 100644
--- a/src/conf/cards/HDA-Intel.conf
+++ b/src/conf/cards/HDA-Intel.conf
@@ -15,6 +15,7 @@ HDA-Intel.pcm.front.0 {
 		slave.pcm {
 			type hw
 			card $CARD
+			subdevice 0
 		}
 		control {
 			name "PCM Playback Volume"
-- 
1.7.6

